.rr.yaml

```diff
@@ -67,8 +68,8 @@ server:
   #
   # Default: <empty map>
   env:
-    - SOME_KEY: "SOME_VALUE"
-    - SOME_KEY2: "SOME_VALUE2"
+    SOME_KEY: "SOME_VALUE"
+    SOME_KEY2: "SOME_VALUE2"
```

Comment:
These were defined as array, but documentation lists them as key-value. I use key-value for my system which works, so I changed these to be key-value also. If both work, it's probably best to just stick to one way and maybe deprecate array of objects in a future release.